docs/HowToGuides/FAQ.md

```diff
@@ -0,0 +1,131 @@
+# Frequently Asked Questions
+
+## Build System and CMake Configuration
+
+### How do I add a new file to CMake?
+
+If you forget to add a new file to the CMake configuration, you may end up with
+undefined symbol errors at link time.
+
+There should be a `CMakeLists.txt` in the directory where you added the new
+file, which has the list of different .h/.cpp files to be included in the build.
+Add your new file to that list.
+
+### How do I speed up iterating on changes to the build system?
+
+The general idea is to build as little as you can.
+- [Use `sccache`](/docs/HowToGuides/GettingStarted.md) or `ccache` if you
+  aren't doing so already.
+- Use `build-script`'s various `--skip-*` flags to skip configuring for
+  platforms that you do not care about.
```

Comment:
Should it be mentioned that not all `--skip-*` flags are documented in `--help`?

Reply:
Do you have some specific flags in mind here? I think the right step here would be to document the flags...

Reply:
https://bugs.swift.org/browse/SR-13531

Reply:
I was under the impression not all skip flags were shown. I don't have a long list or anything, but the ones I have in my shell history are specific to testing, which may void my suggestion (`--skip-test-cmark` and `--skip-test-swift`).

Reply:
All flags should be documented IMO, regardless of whether they are specific to testing or not.

Reply:
agreed